packages/ibmmq/_dev/deploy/variants.yml

```
@@ -0,0 +1,4 @@
+variants:
+  v9.2.4.0-r1-amd64:
+    SERVICE_VERSION: 9.2.4.0-r1-amd64
+default: 9.2.4.0-r1-amd64
```

nit: default will contain variants key name, which is in your case is `v9.2.4.0-r1-amd64`.
```suggestion
  v9.2.4.0-r1-amd64:
    SERVICE_VERSION: 9.2.4.0-r1-amd64
default: v9.2.4.0-r1-amd64
```